templates/functions.php

```diff
@@ -37,6 +37,11 @@ function ap_scripts_front() {
 
 	wp_enqueue_style( 'ap-overrides', ap_get_theme_url( 'css/overrides.css' ), array( 'anspress-main' ), AP_VERSION );
 
+	// Enqueue the TinyMCE JavaScript library file.
+	if ( ap_opt( 'question_text_editor' ) || ap_opt( 'answer_text_editor' ) ) {
```

Comment:
@rahularyan As far as I remember, loading tinyMCE with Ajax will not work on this case since we will need to load it once the page loads for the tinyMCE to work with SyntaxHighlighter addon enabled.